src/config.js

```diff
@@ -36,10 +43,21 @@ class Config {
     this.version = 'v2'
     this.currency = currency
     this.language = language
+
+    let fetchMethod
+    if (custom_fetch && throttleRequests) {
+      fetchMethod = throttleFetch
+    } else if (custom_fetch) {
+      fetchMethod = custom_fetch
+    } else if (throttleRequests) {
+      fetchMethod = throttleFetch
+    } else {
+      fetchMethod = fetch
+    }
```

Comment:
```suggestion
    }
    
```

Reply:
Hi @samblacklock, apologies but I did not get this change. Are we removing `}` and adding `}`? it looks the same thing